cmake/scripts/common/Macros.cmake

```diff
@@ -629,22 +627,29 @@ macro(core_find_versions)
     string(TOLOWER ${APP_VERSION_TAG} APP_VERSION_TAG_LC)
   endif()
   string(REPLACE "." "," FILE_VERSION ${APP_ADDON_API}.0)
-  file(STRINGS ${CORE_SOURCE_DIR}/xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h guilib_version REGEX "^.*ADDON_GLOBAL_VERSION_GUI (.*)$")
-  string(REGEX REPLACE ".*\"(.*)\"" "\\1" guilib_version ${guilib_version})
-  file(STRINGS ${CORE_SOURCE_DIR}/xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h guilib_version_min REGEX "^.*ADDON_GLOBAL_VERSION_GUI_MIN (.*)$")
-  string(REGEX REPLACE ".*\"(.*)\"" "\\1" guilib_version_min ${guilib_version_min})
+
+  # Set defines used on addon.xml.in and readed from versions.h
+  # To set versions about add-on parts automatic
+  # This part is near equal with them on "AddonHelpers.cmake", except the place of versions.h
```

Comment:
"This part is nearly identical to "AddonHelpers.cmake", except location of versions.h" ?